src/PageHeader/PageHeader.tsx

```diff
@@ -57,19 +58,17 @@ const Root: React.FC<React.PropsWithChildren<PageHeaderProps>> = ({children, sx
 // to manage their custom visibility but consumers should be careful if they choose to hide this on narrow viewports.
 // PageHeader.ContextArea Sub Components: PageHeader.ParentLink, PageHeader.ContextBar, PageHeader.ContextAreaActions
 // ---------------------------------------------------------------------
-
 const ContextArea: React.FC<React.PropsWithChildren<PageHeaderProps>> = ({
   children,
   hidden = hiddenOnRegularAndWide,
   sx = {},
 }) => {
-  const isHidden = useResponsiveValue(hidden, false)
   const contentNavStyles = {
-    display: isHidden ? 'none' : 'flex',
     flexDirection: 'row',
     alignItems: 'center',
     gap: '0.5rem',
     order: REGION_ORDER.ContextArea,
+    ...CSSManagedResponsiveValue(hidden, false, 'display', 'none', 'flex'),
```

Comment:
Is the alternative to this helper that the author would need to write something like:

```js
const styles = {};

if (typeof hidden === 'boolean') {
  styles.display = hidden ? 'none' : 'flex';
}

if (typeof hidden === 'object') {
  if (hidden.narrow) {
    styles[breakpoints.narrow] = {
      display: hidden.narrow ? 'none' : 'flex',
    };
  }

  if (hidden.regular) {
    styles[breakpoints.regular] = {
      display: hidden.regular ? 'none' : 'flex',
    };
  }

  if (hidden.wide) {
    styles[breakpoints.wide] = {
      display: hidden.wide ? 'none' : 'flex',
    };
  }
}
```

Reply:
Yes! with the previous version it was exactly this. With the new implementation, it is slightly different but still trying to lift the heavy weight of writing a bunch of condition and media queries.